main.py

```diff
@@ -73,13 +73,18 @@ def _request_flow_list(self, switch):
                                                        'ports': xid_ports}
 
     @listen_to('kytos/of_core.v0x01.messages.in.ofpt_stats_reply')
-    def handle_stats_reply(self, event):
+    def on_stats_reply(self, event):
```

Comment:
Similar [to this comment here](https://github.com/kytos-ng/topology/pull/28#discussion_r769836095) on PR 28 on `topology`, I've refactored the `listen_to` method names to start with `on_*` just so the `handle_*` usage were still the name in terms of testability, let me know if you have other suggestion for this or if we can use this pattern. @italovalcy @ajoaoff @rmotitsuki

Reply:
great suggestion! I agree, @viniarck

Reply:
sounds good, I'll go ahead and merge this PR then, and then for the naming convention I've mapped this one that we've chatted on other PRs https://github.com/kytos-ng/kytos-ng.github.io/issues/12 (adding here the info for future readers), and then we can keep evolving/refining there if any new suggestion or feedback comes up.